src/RequestInsurance/Commands/FailOrReadyProcessingRequestInsurances.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Cego\RequestInsurance\Commands;
+
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+use Cego\RequestInsurance\Enums\State;
+use Cego\RequestInsurance\Models\RequestInsurance;
+use Illuminate\Support\Facades\Log;
+
+
+class FailOrReadyProcessingRequestInsurances extends Command
+{
+    /**
+     * Name and signature of the console command
+     *
+     * @var string
+     */
+    protected $signature = 'fail-or-ready:request-insurances';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Sets requests that have been processing for at least 10 minutes into either failed or ready state';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle() : int
+    {
+       $this->updateProcessingRequestInsurances(true);
+       $this->updateProcessingRequestInsurances(false);
+
+       return 0;
+    }
+
+    /**
+     * Updates request insurances that have been processing for more than 10 minutes and logs it.
+     * Which state they are updated to is based whether it retries inconsistent states.
+     *
+     * @param bool $retries_inconsistent
+     *
+     * @return void
+     */
+    protected function updateProcessingRequestInsurances(bool $retries_inconsistent) : void
+    {
+        $reqs = RequestInsurance::query()->where('state', State::PROCESSING)

```

Comment:
Overvej evt. at hente dem alle sammen ud og bare loop igennem dem:
```php
RequestInsurance::query()->where('state', State::PROCESSING)
->where("state_changed_at", "<", Carbon::now('UTC')->subMinutes(10))
->get()
->each(function(RequestInsurance $requestInsurance) {
/** Opdater dem accordingly / log hver enkelt */
}
```
Så slipper du også for at smide $retriesInconsistent med som parameter til denne metode.